puppet/modules/redmine/manifests/init.pp

```diff
@@ -45,6 +45,15 @@
   }
 
   # Needed for bundle install
+  if $facts['os']['release']['major'] == '8' {
+    package { 'ruby dnf module':
```

Comment:
This be done before `rubygem-bundler` (and why do we list `rubygem-bundler.noarch`?) and `ruby-devel`?

Reply:
it should be, yes. but I didn't know how to tell `ensure_packages` to add a before relationship, and it worked fine w/o

Reply:
You can do a `before` on this package instead of a `requires` on the `ensure_packages`:
```suggestion
    package { 'ruby dnf module':
      before => Package['rubygem-bundler.noarch', 'ruby-devel'],
```